tests/integration/targets/ome_application_network_proxy/tests/ignore_certificate_validation.yaml

```diff
@@ -0,0 +1,130 @@
+# Dell OpenManage Ansible modules
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test that we have an OME host, OME username and OME password
+  ansible.builtin.fail:
+    msg: 'Please define the following variables: hostname, username
+         and password.'
+  when: hostname is not defined or username is not defined or password
+         is not defined
+
+- block:
+    # PRE REQ - CONFIGURE NETWORK PROXY
+    - name: Reset proxy configuration.
+      dellemc.openmanage.ome_application_network_proxy:
+        enable_proxy: false
+
+    - name: Update ignore_certificate_validation. (check mode - changes expected)
```

Comment:
Limit the line length to 80 character. Make this changes at all applicable places

Reply:
updated